### DISCUSSION TITLE
Help: Simple 2 Key Macro

### DISCUSSION THREAD

**Discussion by tklvan** [community/community#147195] [Programming Help]
### Select Topic Area

Question

### Body

Hi. I’m brand new to coding/programing. I’m specifically using python and I was wondering if someone could explain to me how to build a very specific macro. The macro will press 1 key for a certain amount of time then alternate to another key for a certain amount of time in an infinite loop.

My plan was to create this macro then implement it into a Raspberry PI controller or something like that which will then be connected to a prototype physical key pressing device if this makes any sense haha.

**Comment by shinybrightstar** [comment]
Welcome to the GitHub Community, @tklvan , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it to Programming Help for you. Good luck!
